tools/tck-regression/README.adoc

```diff
@@ -0,0 +1,98 @@
+= TCK Regression
```

Comment:
I find the name "Regression" for this module confusing. A regression is "a return to a former or less developed state." What this module does is comparison reports that indicate regressions or improvements.

Reply:
Agree. Do you think "TCK Report Comparison" is more reasonable? (Will rename module as well)

Reply:
That sounds reasonable.